tox.ini

```diff
@@ -16,7 +16,9 @@ commands =
   py.test
 
 [testenv:docs]
-deps = collective.checkdocs
+deps =
+    collective.checkdocs
+    pygments
```

Comment:
Why do we need pygments?

Reply:
The `pygments` library is required for syntax highlighting.

Reply:
Ah, I see, thanks!